app/src/main/java/nu/yona/app/utils/AppUtils.java

```diff
@@ -398,6 +399,14 @@ private static AlertDialog getGenericAlertDialogWithErrorMessage(ErrorMessage er
 		return alertDialogBuilder.create();
 	}
 
+	@TargetApi(Build.VERSION_CODES.O)
+	public static boolean arePersistentNotificationsEnabled(Context context, String channelId)
+	{
+		android.app.NotificationManager notificationManager = context.getSystemService(android.app.NotificationManager.class);
+		NotificationChannel notificationChannel = notificationManager.getNotificationChannel(channelId);
+		return notificationChannel != null && notificationChannel.getImportance() == android.app.NotificationManager.IMPORTANCE_NONE;
```

Comment:
``notificationChannel`` can only be null when an invalid ``channelId`` is given, right? If that's true, then this condition should be changed into:
```
		return notificationChannel.getImportance() == android.app.NotificationManager.IMPORTANCE_NONE;
```
It will cause an NPE when the ID is wrong, thus exposing that issue.

Reply:
At the very beginning of the application, the notificationChannel can be null on the first install of the application, which will cause a NPE here. So I believe null check is needed. I will test the hotfix branch and confirm it.

Reply:
👍

Reply:
It causes a crash on first install on my device. As the channel with said ID is not yet created by this time. I will add the check back.

Reply:
👍 
Please add a comment explaining the reason, so we don't remove it again as unnecessary.

Reply:
Adding this null check creates another issue. This will cause alert to popup even on first launch even before user is aware of what the channel is about. 
I believe silently creating the channel is better solution. And use this channel when needed with the ID.

Reply:
That makes sense indeed.